docs/howto/gdnative.md

```diff
@@ -281,7 +281,7 @@ In a text editor, create a file called ***steam.gd*** and place the following in
 ```gdscript
 extends Node
 
-onready var Steam: Object = preload("res://addons/godotsteam/godotsteam.gdns").new()
+@onready var Steam: Object = preload("res://addons/godotsteam/godotsteam.gdns").new()
```

Comment:
This one is slightly incorrect, I think.  Godot 3 does not have the @ symbol in front of onready but Godot 4 does.  Yeah, the Godot 3 editor considers adding the @ as an error.

Reply:
Gotcha, I'll revert